docs/CHANGELOG.asciidoc

```diff
@@ -66,6 +66,19 @@ Review important information about the {kib} 8.x releases.
 
 For information about the {kib} 8.13.0 release, review the following information.
 
+[float]
+[[known-issues-8.13.0]]
+=== Known issues
+
+[discrete]
+[[known-179457]]
+.Canvas workpad image rendering displays a fail toast notification
+[%collapsible]
+====
+*Details* +
+When opening an ecommerce sample data in a Canvas workpad, a toast notification stating that rending the image has failed is incorrectly displayed. For more information, refer to ({kibana-pull}179457[#179457]).
```

Comment:
```suggestion
When opening an ecommerce sample data in a Canvas workpad, a toast notification stating that rendering the image has failed is incorrectly displayed. For more information, refer to ({kibana-pull}179457[#179457]).
```

Apart from the typo, maybe @nickpeihl or @cqliu1 can take a stab at improving the note as per @bhavyarm recommendation?

My attempt would be something like:

```
In Canvas, a revealed image component displays a wrong error in a toast, stating that the image rendering failed. For more information, refer to ({kibana-pull}179457[#179457]).
```

Reply:
Oh dear, I made a bad assumption about it just being the sample data. Thank you for catching that!

Reply:
I dug into this a bit more and I don't think it is related to the revealImage element type. It appears to happen when any markdown element is added to the workpad, but there is no content. A fix was added in 8.14. https://github.com/elastic/kibana/pull/177989

I exported the sample data workpad and found that the black lines are actually empty markdown elements with a border.

So, I think the known issue should say something like


`
In Canvas, an empty Text element incorrectly displays a "Markdown content is required in [readOnly] mode" error in a pop-up notification. For more information, refer to ({kibana-pull}177989[#177989]).
`

Reply:
I've currently gone with:

`In Canvas, and empty text element incorrectly triggers a "Markdown content is required in [readOnly] mode" toast notification. For more information, refer to ({kibana-pull}179457[#179457]).` 

Hope that's ok :)